src/enumerate.rs

```diff
@@ -642,3 +642,436 @@ impl UserProfile {
         }
     }
 }
+
+#[pyclass]
+pub(crate) struct CoreEvent(libparsec::core::CoreEvent);
+
+#[pymethods]
+impl CoreEvent {
+    #[classattr]
+    #[pyo3(name = "BACKEND_CONNECTION_CHANGED")]
+    fn backend_connection_changed() -> &'static PyObject {
```

Comment:
Maybe we can have a macro now to generate these block

Reply:
Sure, I've used vim macros myself to generate this code.

Reply:
Done.

Reply:
I thought about a generic macro which can be used by others :thinking:

Reply:
I modified the macro to make it more generic. I think we should use this enum in an other PR.